src/runtime/CMakeLists.txt

```diff
@@ -163,6 +165,7 @@ set(RUNTIME_CXX_FLAGS
     -Wno-psabi
     -Wno-unknown-warning-option
     -Wno-unused-function
+    -Wno-atomic-alignment
```

Comment:
Why is this required? Where are we getting the warnings? (This sounds like a warning we shouldn't be ignoring!)

Reply:
Ah, this vaguely rings a bell, but not gonna investigate tonight. Definitely flag it with a `# TODO: explain the issue here` comment so we don't overlook circling back to it.

Reply:
See src/runtime/synchronization_common.h. It wraps a lot of these atomic ops to avoid that error. See also src/runtime/scoped_spin_lock.h

Reply:
Thanks for the pointers @abadams, I've tweaked my atomic usage and remove `-Wno-atomic-alignment`.